sustainability/src/main/java/edu/asu/diging/sustainability/core/service/impl/ConceptManager.java

```diff
@@ -66,4 +66,18 @@ public void storeConceptSearchCategories(String conceptId, List<SearchCategory>
         concept.setSearchCategories(searchCategories);
         conceptRepo.save((Concept) concept);
     }
+
+    /**
+     * Store the alias for concept name for the concept with given Id.
+     * 
+     * @param conceptId The id of the concept for which the alias is provided.
+     * @param alias Alias for the concept name to be saved.
+     */
+    @Override
+    public void storeConceptAlias(String conceptId, String alias) {
```

Comment:
merge with 'storeConceptSearchCategories'. no reason there would be an extra method for bot properties if there could just be one update method.

Reply:
also, let's write some tests at least for the update method.